pkg/sqlmigrations/migrations.go

```diff
@@ -512,3 +553,46 @@ func addRootUser(ctx context.Context, r runner) error {
 	}
 	return err
 }
+
+func addRoles(ctx context.Context, r runner) error {
+	// System tables can only be modified by a privileged internal user.
+	session := r.newRootSession(ctx)
+	defer session.Finish(r.sqlExecutor)
+
+	// Add the roles column to the system.users table.
+	const alterStmt = `
+		ALTER TABLE system.users ADD COLUMN IF NOT EXISTS "isRole" BOOL NOT NULL DEFAULT false;
+		 `
```

Comment:
Don't think you need the `;` here.

Reply:
Done here and the rest of `migrations.go`. We seem inconsistent about doing that.